sdk/src/main/java/io/radar/sdk/RadarVerificationManager.kt

```diff
@@ -91,6 +95,10 @@ internal class RadarVerificationManager(
                         ) {
                             if (status != Radar.RadarStatus.SUCCESS || location == null) {
                                 Radar.handler.post {
+                                    if (status != Radar.RadarStatus.SUCCESS) {
```

Comment:
Nit on the above `location == null` check: if `location == null` necessarily `status != Radar.RadarStatus.SUCCESS` so the `location == null` check is redundant` and then this inner `status != Radar.RadarStatus.SUCCESS` is redundant.